### ISSUE TITLE
RESONITE STREAM OUTPUT

### ISSUE THREAD

**Issue by ExoTheWicker** [Yellow-Dog-Man/Resonite-Issues#777]
### Is your feature request related to a problem? Please describe.

I've on multiple occasions run into problems where my friends or I  would be streaming to a service like YouTube but have the problem where we have to mute multimedia because someone was playing copyrighted music

### Describe the solution you'd like

I would like to suggest a setting for the new settings UI (no it's not motion blur). I think it would be nice to have a "RESONITE STREAM OUTPUT" to which users can assign audio tracks like voice and UI and unassign music so streamers can stream worlds with music without worrying about DMCA

### Describe alternatives you've considered

just muting multimedia in resonite while streaming which will take away from the experience.

**Comment by Lexevolution** [comment]
Just to make sure I understand this properly, are you asking for a way to route the different audio categories to different audio outputs?

**Comment by ExoTheWicker** [comment]
> Just to make sure I understand this properly, are you asking for a way to route the different audio categories to different audio outputs?

More like having an output that can be told what it has like voice and UI but no music

**Comment by epicEaston197** [comment]
Why can't we just separate multimedia into video in music?
Wouldn't that solve this issue?

**Comment by 5H4D0W-X** [comment]
It wouldn't, as videos can still contain copyrighted audio. Having a streaming audio output that shows up as a mic would be the best solution

**Comment by troyBORG** [comment]
Yeah I would like the thing where each thing is a separate audio source.

I use to use this when I streamed Xbox, where I had My Microphone, Game Audio, Party Chat, and Music as all separate audio tracks.  And could mute different tracks to the stream but still hear it myself.

Or being able to after the fact open the local recording and be able to censor audio (or remove sneeze/burps/misc noises or audio) that was said in party chat that I don't need in a YouTube Video.
